packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.mjml

```diff
@@ -1,33 +1,33 @@
-<%# This Source Code Form is subject to the terms of the Mozilla Public 
-  # License, v. 2.0. If a copy of the MPL was not distributed with this 
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
   # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
 
 <mj-include path="./css/appBadges/index.css" type="css" css-inline="inline" />
 
-<% locals.iosUrl = locals.iosUrl || locals.appStoreLink %> 
-<% locals.androidUrl = locals.androidUrl ||  locals.playStoreLink %> 
+<% locals.iosUrl = locals.iosUrl || locals.appStoreLink %>
+<% locals.androidUrl = locals.androidUrl ||  locals.playStoreLink %>
 
 <mj-section>
   <mj-group css-class="app-badges">
 
     <% if (locals.iosUrl) { %>
       <mj-column>
         <mj-image
-          css-class="app-badge"
+          css-class="app-badge-ios"
```

Comment:
This `css-class`has been updated to utilize within `mj-html-attributes`.

Reply:
If this is because we need the selector to be unique, can we use IDs instead of unique classes with the same styles?

Reply:
I don't believe so - per [mjml documentation](https://documentation.mjml.io/?html#mj-html-attributes), the `mj-html-attributes` tag allows us to add custom attributes on any html tag of the generated html, using css selectors.

Reply:
Does it let you use the css selector `#app-badge-ios` and `#app-badge-android`? I think we should just use IDs like that then instead of creating extra classes if that's possible, it is less confusing that way IMO.